src/store/__mocks__/load-data.js

```diff
@@ -2,6 +2,7 @@ import { getUrl } from '../../utils';
 import animals from '../../utils/data/animals.mock.json';
 import demo from '../../utils/data/demo.mock.json';
 import node_task from '../../utils/data/node_task.mock.json';
+import node_plot from '../../utils/data/node_plot.mock.json';
```

Comment:
I'm not sure why these are not already using camel case, but would you mind updating all of them for consistency if possible?